frontend/src/marketplace/EntryList.tsx

```diff
@@ -0,0 +1,51 @@
+import { Box, Link as MuiLink, Typography } from '@mui/material'
+import { useTheme } from '@mui/material/styles'
+import { ModelSearchResult } from 'actions/model'
+import Link from 'next/link'
+import { Fragment } from 'react'
+import ChipSelector from 'src/common/ChipSelector'
+import EmptyBlob from 'src/common/EmptyBlob'
+
+interface EntryListProps {
+  entries: ModelSearchResult[]
+  selectedChips: string[]
+  handleChipsOnChange: (chips: string[]) => void
```

Comment:
This is effectively an event prop as we are just passing it through, so consider renaming to `onSelectedChipsChange`.

I believe this is the agreed upon convention. `on...` denotes an event, `handle...` denotes a handler for an event.